packages/flutter/lib/src/material/theme.dart

```diff
@@ -20,6 +21,7 @@ class Theme extends InheritedWidget {
     assert(data != null);
   }
 
+/// Holds attribute values that specify the look and format of this widget.
```

Comment:
Bad indent.  Also, I'd skip the introductory phrasing.  Perhaps:  "Specifies the color and typography for descendant widgets."